src/guide/built-ins/keep-alive.md

```diff
@@ -14,7 +14,7 @@ In the Component Basics chapter, we introduced the syntax for [Dynamic Component
 <component :is="activeComponent" />
 ```
 
-By default, an active component instance will be unmounted when switched away from. This will cause any changed state it holds to be lost. When this component is displayed again, a new instance will be created with only the initial state.
+By default, an active component instance will be unmounted when switched away from it. This will cause any changed state it holds to be lost. When this component is displayed again, a new instance will be created with only the initial state.
```

Comment:
I think we need to change the verb slightly too:

```suggestion
By default, an active component instance will be unmounted when switching away from it. This will cause any changed state it holds to be lost. When this component is displayed again, a new instance will be created with only the initial state.
```

Reply:
@skirtles-code good point.

Reply:
If you'd like to accept my proposed change you can merge it into your branch by clicking the 'Commit suggestion' button beneath the change.